tests/projects/c++/modules/stdmodules/xmake.lua

```diff
@@ -3,7 +3,8 @@ set_languages("c++latest")
 
 target("mod")
     set_kind("static")
-    add_files("src/*.cpp", "src/*.mpp")
+    add_files("src/*.cpp")
+    add_files("src/*.mpp", {public = true})
     set_policy("build.c++.clang.stdmodules", true)
```

Comment:
it has been renamed.

Reply:
what has been renamed ?

Reply:
warning: unknown policy(build.c++.clang.stdmodules), please run `xmake l core.project.policy.policies` if you want to all policies